.ci/openshift-ci/build-root/e2e-test.sh

```diff
@@ -50,6 +50,7 @@ xtf.openshift.master.kubeconfig=${KUBECONFIG}
 intersmash.wildfly.operators.catalog_source=community-operators-wildfly-operator
 intersmash.wildfly.operators.index_image=quay.io/operatorhubio/catalog:latest
 intersmash.wildfly.operators.package_manifest=wildfly
+intersmash.wildfly.operators.channel=alpha
```

Comment:
not related, but I'd replace `mvn clean test` with `mvn test` on line 60

Reply:
Thanks @tommaso-borgato! will give it a try right now!